react/features/participants-pane/components/FooterContextMenu.js

```diff
@@ -1,19 +1,28 @@
 // @flow
 
 import { makeStyles } from '@material-ui/core/styles';
+import clsx from 'clsx';
```

Comment:
If we have a new dependency we need to declare it in package.json?

Reply:
My bad, got removed in the rebase